opam-packages-conversion/convertedPackages.txt

```diff
@@ -1,6 +1,6 @@
 # Forks
 ./forks/camlp4 esy/4.02+7 4.02+7
-./forks/ocamlfind esy/1.7.1 origin/master
+./forks/ocamlfind origin/esy/1.7.1 origin/master
```

Comment:
This is the thing I've updated. I've updated the `package.json` in the ocamlfind fork (you can see I force pushed already). The update I've made is compatible with *either* escaping scheme so it's safe for current users of `esy`. I've tested.